services/slam/fake/slam_test.go

```diff
@@ -267,12 +116,10 @@ func reverse[T any](slice []T) []T {
 	return slice
 }
 
-func verifyGetPointCloudMapStreamStateful(t *testing.T, slamSvc *SLAM, extra map[string]interface{}) {
+func verifyGetPointCloudMapStreamStateful(t *testing.T, slamSvc *SLAM) {
 	testDataCount := maxDataCount
 	getPointCloudMapResults := []float64{}
-	positionResults := []spatialmath.Pose{}
 	getPositionResults := []spatialmath.Pose{}
-	getInternalStateResults := []int{}
 	getInternalStateStreamResults := []int{}
```

Comment:
[nit] update names to final version to avoid having to rename in future PR `getInternalStateResults`